src/helpers/order-process/order-process-helper.ts

```diff
@@ -73,8 +89,47 @@ export const removeOrderProcesses = (
     )
   );
 
-export const updateOrderProcess = (
+export const updateOrderProcess = async (
   id: string,
-  data: Partial<OrderProcess>
-): Promise<OrderProcess> =>
-  getOrderProcessApi().updateOrderProcess(id, data) as Promise<OrderProcess>;
+  {
+    before_portfolio_item_id,
+    after_portfolio_item_id,
+    ...data
+  }: Partial<OrderProcess>
+): Promise<OrderProcess> => {
+  const op = await getOrderProcessApi().updateOrderProcess(id, {
+    name: data.name,
+    description: data.description,
+    before_portfolio_item_id: '',
+    after_portfolio_item_id
+  });
+
+  return op as OrderProcess;
+};
+
+export const addOrderProcess = async ({
+  before_portfolio_item_id,
+  after_portfolio_item_id,
+  ...data
+}: Partial<OrderProcess>): Promise<OrderProcess> => {
+  const op = await getOrderProcessApi().createOrderProcess({
+    name: data.name,
+    description: data.description
+  });
+
+  const promiseB =
+    before_portfolio_item_id !== undefined
+      ? getOrderProcessApi().addOrderProcessBeforeItem(
+          ((op as unknown) as OrderProcess).id as string,
+          { portfolio_item_id: before_portfolio_item_id }
+        )
+      : {};
+  const promiseA =
+    after_portfolio_item_id !== undefined
+      ? getOrderProcessApi().addOrderProcessAfterItem(
+          ((op as unknown) as OrderProcess).id as string,
+          { portfolio_item_id: after_portfolio_item_id }
+        )
+      : {};
+  return Promise.all([promiseA, promiseB]) as Promise<OrderProcess>;
```

Comment:
This function surely does not return a promise with a single order process. Promise all always returns an array of responses in the same order as the in the array. This will require a custom type. IF the `addOrderProcessBeforeItem` returns a OrderProcess, then it should look something like this: `Promise<[OrderProcess, OrderProcess]>`